packages/core/src/helpers/rule.ts

```diff
@@ -47,8 +48,13 @@ export function createSubsetAst<T extends postcss.Root | postcss.AtRule>(
     const prefixType = prefixSelectorList[0].nodes[0];
     const containsPrefix = containsMatchInFirstChunk.bind(null, prefixType);
     const mixinRoot = mixinTarget ? mixinTarget : postcss.root();
-    root.nodes.forEach((node) => {
-        if (node.type === `rule` && (node.selector === ':vars' || node.selector === ':import')) {
+    [...root.nodes].forEach((node) => {
```

Comment:
why do we clone the here if you clean each append?

Reply:
removed